.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Package and Push
+
+on:
+  # Triggers the workflow on pull request events and merges/pushes to main
+  push:
+    branches:
+      - master
+      - main
+
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+  schedule:
+    # Daily 5am australian/brisbane time
+    - cron: '0 19 * * *'
+
+jobs: 
+  build:
+    runs-on: ubuntu-latest
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true # to allow setting version env var
+      VERSION: "1.1.${{ github.run_number }}"
```

Comment:
🤔 This was the simplest way I could find to increment a build. 

The only problem I think is that this never resets. I've used 1.1.x since thats what version is listed in team city